ochre/Models/Water.py

```diff
@@ -48,6 +49,7 @@ class StratifiedWaterModel(RCModel):
         'Zone Temperature (C)',
     ]
 
+    # Starting with 1 PCM node
```

Comment:
What we really need here are nodes that are partially water and partially PCM. Getting a PCM model in is useful, but it's going to be very hard to test if you only allow nodes to be 100% PCM or 100% water.